src/SignalR/common/Protocols.MessagePack/src/MessagePackHubProtocolOptions.cs

```diff
@@ -9,24 +9,31 @@ namespace Microsoft.AspNetCore.SignalR
 {
     public class MessagePackHubProtocolOptions
     {
-        private IList<IFormatterResolver> _formatterResolvers;
+        private MessagePackSerializerOptions _messagePackSerializerOptions;
 
-        public IList<IFormatterResolver> FormatterResolvers
+        /// <summary>
+        /// <para>Get or Set the <see cref="MessagePackSerializerOptions"/> used internally by the <see cref="MessagePackSerializer" />.</para>
+        /// <para>If you override it, we strongly recommend that you set <see cref="MessagePackSecurity" /> to <see cref="MessagePackSecurity.UntrustedData"/> by calling :<para>
+        /// <code>customMessagePackSerializerOptions.WithSecurity(MessagePackSecurity.UntrustedData)</code>
```

Comment:
```suggestion
        /// <code>customMessagePackSerializerOptions = customMessagePackSerializerOptions.WithSecurity(MessagePackSecurity.UntrustedData)</code>
```

Reply:
this is better than my previous summary but is it explicit enough for consumer that they will have to affect it back like
`options.SerializerOptions = customMessagePackSerializerOptions.WithSecurity(MessagePackSecurity.UntrustedData)`

or

```csharp
var customMessagePackSerializerOptions = MessagePackSerializerOptions.WithXXXXX // Your custom MsgPack Serializer Options
customMessagePackSerializerOptions = customMessagePackSerializerOptions.WithSecurity(MessagePackSecurity.UntrustedData)
options.SerializerOptions = customMessagePackSerializerOptions;
```

I'm not sure if the summary is going to be too big 
or if the priority is to be sure that consumer could have a full understanding just by reading it

Reply:
welllll
I should remove the `,` since the `<code>` seems to create a paragraph right 
@anurse  ?

![image](https://user-images.githubusercontent.com/2266487/78290507-1cc27580-7524-11ea-8c66-d0b9ad250064.png)